src/Authenticator/CookieAuthenticator.php

```diff
@@ -51,6 +52,7 @@ class CookieAuthenticator extends AbstractAuthenticator implements PersistenceIn
             'name' => 'CookieAuth',
         ],
         'passwordHasher' => 'Authentication.Default',
+        'use_salt' => true,
```

Comment:
Please use camelBacked syntax for option keys.

Reply:
Of course, thanks.